src/components/ImportJobs/components/ReturnToAssignJobs/ReturnToAssignJobs.js

```diff
@@ -24,11 +27,16 @@ class ReturnToAssignJobs extends Component {
 
   render() {
     const { deleteUploadDefinition } = this.context;
+    const { prohibitFilesUploading } = this.props;
+
+    const messageId = prohibitFilesUploading
```

Comment:
I think it can be used only one line for this statement instead of three.

```suggestion
    const messageId = prohibitFilesUploading ? 'messageWhenProhibited' : 'message';
```